### PR TITLE
generate swagger output for streaming endpoints with a basic note

### PR DESCRIPTION
**Pull request by tmc**

### CODE REVIEW

**Comment by yugui** [comment]
LGTM, but could you update the generated files as well?

File: protoc-gen-swagger/genswagger/template.go [hunk 1]
```diff
@@ -387,13 +389,17 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 				}
 
 				methProtoPath := protoPathIndex(reflect.TypeOf((*pbdescriptor.ServiceDescriptorProto)(nil)), "Method")
+				desc := ""
+				if meth.GetServerStreaming() {
+					desc += "(streaming responses)"
```
Comment (achew22) [inline protoc-gen-swagger/genswagger/template.go]: Does this make swagger generate a streaming client?

Comment (tmc) [inline protoc-gen-swagger/genswagger/template.go]: @achew22 swagger spec doesn't address streaming endpoints (yet) there's some discussion on the openapi repo.

**Comment by tmc** [comment]
@yugui ping

**Pull request merged**